GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -136,6 +137,25 @@ public void setScript(GVRScript gvrScript, String distortionDataFileName) {
         }
     }
 
+    /**
+     * Sets whether to force rendering to be single-eye, monoscopic view.
+     *
+     * @param force
+     *            If true, will create a GVRMonoscopicViewManager when {@linkplain setScript setScript()} is called.  If false, will proceed to auto-detect whether the device supports VR rendering and choose the appropriate ViewManager.  This call will only an effect if it is called before {@linkplain setScript setScript()}.
```

Comment:
{@link syntax is off, and text is not auto-formatted.

     *            If true, will create a GVRMonoscopicViewManager when
     *            {@linkplain #setScript(GVRScript, String) setScript()} is
     *            called. If false, will proceed to auto-detect whether the
     *            device supports VR rendering and choose the appropriate
     *            ViewManager. This call will only have an effect if it is
     *            called before {@linkplain #setScript(GVRScript, String)
     *            setScript()}.